### ISSUE TITLE
babel config conflict

### ISSUE THREAD

**Issue by CharlesOkwuagwu** [alessiomaffeis/vue-picture-input#57]
I'm trying to migrate a project from vue-cli 2.x to 3.0, every thing works fine except this module for which i get this error:

```
 ERROR  Failed to compile with 1 errors                                                                                                                                                                                                10:10:46

 error  in ./node_modules/vue-picture-input/PictureInput.vue

Module build failed: Error: Cannot find module 'babel-preset-env' from 'C:\webdev\rsg4\node_modules\vue-picture-input'
- Did you mean "@babel/env"?

```
A suggested fix from @LinusBorg https://forum.vuejs.org/t/errors-moving-to-vue-cli-3-0/29397/6

> remove the babel config from their npm package by adding it to .npmignore

Will the suggested workaround be possible.

Thanks in advance.

Charles.

**Comment by alessiomaffeis** [comment]
Hi Charles,

I just added an .npmignore file and published a new version (2.1.5) to npm.

That should fix the error. Let me know if it works! :)

**Comment by CharlesOkwuagwu** [comment]
Works perfectly!
